frontend/admin/src/pages/purchase/PurchaseDetail.tsx

```diff
@@ -61,6 +61,17 @@ const PurchaseDetailInternal = (props: any) => {
             .catch();
     };
 
+    const deletePurchase = () => {
+        if (window.confirm('Möchtest du diesen Einkauf wirklich löschen?')) {
+            purchaseAPI.delete(purchase.uuid)
+                .then(() => {
+                    setShowCompleteToast(true)
+                })
+                .catch();
+            window.location.href = routes.purchaseList();
```

Comment:
Inside react components I'd use the react router, but this works as well.

Router example:
1. Define history variable (in component scope) `const history = useHistory()`
2. Add redirect using `history.push(routes.purchaseList());`

Reply:
Also, you should probably only redirect if the `delete` request was succesful, so move this up to line 68